src/components/common/DocumentDelivery.vue

```diff
@@ -1,39 +1,39 @@
 <template>
-  <v-card flat class="px-5 py-6" id="document-delivery">
```

Comment:
The classes are now specified centrally in the parent component. This will make sense when you look at the SCSS in the parent component.

The other classes changed here as the same. (Also in a few other sub-components.)